semconv/v1.26.0/attribute_group.go

```diff
@@ -1276,7 +1276,7 @@ const (
 	// Stability: experimental
 	// Examples: 'arn:aws:lambda:REGION:ACCOUNT_ID:function:my-function',
 	// '//run.googleapis.com/projects/PROJECT_ID/locations/LOCATION_ID/services/SERVICE_ID',
-	// '/subscriptions/<SUBSCIPTION_GUID>/resourceGroups/<RG>/providers/Microsoft.Web/sites/<FUNCAPP>/functions/<FUNC>'
+	// '/subscriptions/<SUBSCRIPTION_GUID>/resourceGroups/<RG>/providers/Microsoft.Web/sites/<FUNCAPP>/functions/<FUNC>'
```

Comment:
This (and all all the other fixes in semantic conventions, except for the tests) are auto-generated, and need to be fixed in the semantic conventions repository in addition to here (if they haven't yet), or the typo will come back with the next semconv release.

https://github.com/search?q=repo%3Aopen-telemetry%2Fsemantic-conventions+SUBSCIPTION_GUID&type=code

Reply:
Good to know, will followup upstream...

Reply:
Accepted - https://github.com/open-telemetry/semantic-conventions/pull/1379